source/summary.rst

```diff
@@ -38,40 +51,54 @@ already implemented; we link to the repositories in the appropriate places.
 Attack model and terminology
 ++++++++++++++++++++++++++++
 
-We consider a *network adversary* which can access and modify network messages.
-Examples of this adversary can be an ISP, an e-mail provider, an AS,
+We consider a *network adversary* that can read, modify, and create
+network messages.
+Examples of such an adversary are an ISP, an e-mail provider, an AS,
 or an eavesdropper on a wireless network.
-The goal of the adversary is to perform a machine-in-the-middle attack
-on the key exchange within peers. This would enable her to
-i) read the content of messages, and ii) impersonate peers.
-We presume that all peers are honest. They do not collaborate with the adversary and follow the protocols described in this document.
-
-Peers need access to a *trusted*, *out-of-band* channel of communication
-that is not visible to the adversary, and thus cannot be manipulated.
-This channel can be used
-from end-user key verification or key authentication protocols,
-without fear of intervention by the adversary.
-
-Problems of current key verification techniques
+The goal of the adversary is to i) read the content of messages, ii)
+impersonate peers -- communication partners, and iii) to learn who communicates
+with whom.
+To achieve these goals,
+an active adversary might try, for example,
+to perform a machine-in-the-middle attack on the key exchange protocol
+between peers.
+
+Peers learn the content of the messages.
+Therefore,
+we assume that all peers are honest.
+They do not collaborate with the adversary and follow the protocols described in this document.
+
+To enable secure key-exchange and key-verification between peers,
+we assume that peers have access to a *trusted*, *untappable*, *out-of-band*
+communication channel that is not visible to the adversary,
+and thus cannot be manipulated.
+
+Problems of current key-verification techniques
 +++++++++++++++++++++++++++++++++++++++++++++++
 
-In existing e2e-encrypting messengers
+An important aspect of secure end-to-end (e2e) encryption is the verification of
+a peer's key.
+In existing e2e-encrypting messengers,
 users perform key verification by triggering two fingerprint verification workflows:
-each of the two peers show and read the other's key fingerprint data
+each of the two peers shows and reads the other's key fingerprint
 through a trusted channel (often a QR code show+scan).
 
 We observe the following issues with these schemes:
 
-- The scheme requires that both peers start the verification workflow to assert
+..
+  TODO: I'm not sold on the second argument. I think the problem is that to _join_
+  the group, I must verify ``N`` times.
```

Comment:
I think the two are equivalent. Two construct the full group you need :math:`N(N-1) / 2` verificatoins:
1 for the initial pair, then two for the next person joining and N-1 for the last person. So the total is :math:`N(N-1) / 2`. 

Your way of phrasing it seems better to me because we are also describing the new protocol in terms of a person joining the group.